apps/system/js/devtools/logshake.js

```diff
@@ -196,12 +202,16 @@
           return navigator.mozL10n.get('logsOperationFailed',
                                        { operation: error.operation });
         }
+
+        if ('name' in error && error.name === 'NotFoundError') {
+          return navigator.mozL10n.get('logsNotFoundError');
+        }
       }
 
       return '';
     },
 
-    showErrorMessage: function(error, notif) {
+    showErrorMessage: function(title, error, notif) {
```

Comment:
Is it possible to protect the user from displaying multiple times the error message? Example: If you triple tap the notification, you currently have to dismiss 3 error messages.